### ISSUE TITLE
Add GDPR notice

### ISSUE THREAD

**Issue by orbeckst** [MDAnalysis/MDAnalysis.github.io#133]
As part of #129 and following after #131 (switch to GoatCounter) we should add a GDPR notice.

**Comment by orbeckst** [comment]
No collection of personally identifiable information so for right now we won't do more than what we have in the privacy notice. Will reopen if situation changes.
